docs/pages/installation/reverse-proxy-configuration.rst

```diff
@@ -0,0 +1,169 @@
++================+======================+==================+===========+===========================+===========================+
+| nuts endpoints | none/local generated | nuts development | nuts demo | nuts test                 | nuts production           |
++----------------+----------------------+------------------+-----------+---------------------------+---------------------------+
+| data endpoints | none                 | nuts development | nuts demo | PKIo with acceptance list | PKIo with acceptance list |
++----------------+----------------------+------------------+-----------+---------------------------+---------------------------+
+
+Nuts endpoints cover all API's provided by Nuts nodes. Data endpoints are provided by the vendor and cover patient related data.
+The **local**, **development** and **demo** roots follow the same convention for both the nuts endpoints and the data endpoints.
+Since no real data will be used within these environments, using the different Nuts root certificates can simplify deployment.
+
+For **test** and **production** this is a different story.
+A national recognized trusted root in combination with specific trusted certificates gives maximum security.
+More on the acceptance list can be read below. The different PKIo roots can not be obtained from the Nuts API's.
+
+The different Nuts root certificates will be published on https://nuts.nl
+A full Nuts chain would consist of a root, a Nuts intermediate and multiple vendor CAs.
+
+The ``/api/mtls/*`` APIs on :ref:`nuts-registry-api` describe how to obtain a list of CAs in various formats.
+Most reverse-proxies do not support calling API's but only file based configuration.
+It's up to the vendor to convert the API output to a file and restart the proxy when something has changed.
+A single ``truststore.pem`` file with all CAs in PEM format is available in the directory configured by the ``nuts.crypto.fspath`` variable.
+This file can be synced or linked to from various configs.
+
+Acceptance list
+===============
+
+All client certificates used for mTLS will be published in the Nuts registry.
+This will allow for extracting the list of certificates that will be used to the application level.
+Vendors could use this information to enable or disable certain certificates on demand.
+This means that vendors will have full control over which certificate is granted access and which isn't.
+
+Most reverse-proxies require a configured list of CAs to enable mTLS.
+Most of the time, however, they will not be able to configure a list of accepted non-CA certificates.
+Specific certificate acceptance must therefore be done at the application level.
+
+The ``/api/mtls/*`` APIs on :ref:`nuts-registry-api` describe how to obtain an acceptance list in various formats.
+
+Examples
+********
+
+Below are configuration examples on how to configure various reverse proxies.
```

Comment:
This guide applies to Nuts Network (gRPC) as well, right? If so, we'll probably need an issue for that project to support TLS termination on proxies. These examples also don't really reflect that, I think.

Reply:
nope, gRPC is currently not part, it's an implementation detail

Reply:
Maybe we need to make it clear whether this guide is for securing data endpoints, nuts endpoints or both?